renderdoc/driver/vulkan/vk_postvs.cpp

```diff
@@ -56,6 +56,21 @@ static void ConvertToMeshOutputCompute(const ShaderReflection &refl, const SPIRV
 {
   SPIRVEditor editor(modSpirv);
 
+  // strip away any execution modes from the original
+  for (SPIRVIterator it = editor.BeginAll(); it; it++) {
+     if (it.opcode() == spv::OpExecutionMode) {
```

Comment:
Removing all execution modes unconditionally is not always the right thing to do. We don't know what the execution mode is and it may be necessary to run correctly - which means either we need to translate it/process it for compute, or it may be something that's valid for all shader types.

From your message on IRC it sounded like you maybe had multiple entry points in your SPIR-V module and since this patching process removed all other entry points you ended up with a dangling `OpExecutionMode` pointing to an invalid ID. That's a narrower definition of things we need to strip - `OpExecutionMode` entries that point to other entry points.